docs/index.rst

```diff
@@ -19,11 +19,13 @@ Catkit2
    :maxdepth: 1
    :caption: Built-In Services
 
-   service/aimtti_plp
+   services/aimtti_plp
+   services/allied_vision_camera
    services/bmc_dm
    services/camera_sim
```

Comment:
The dummy_camera service is missing here too, I don't know is this is wanted or not

Reply:
Later the safety_manual_check is missing too

Reply:
True again! I have to admit I forgot what the dummy camera service is - it looks like a stand-in service for a camera that is simpler than camera_sim. Since I don't know of any active usage anywhere, or even a reason for it, I think it's ok to not add it for now. What are your thoughts?

Reply:
As for the Safety Manual Check, good catch! I didn't find it because it doesn't even have a doc page yet. I added an empty one now and added it to the list of doc pages that need to be filled in, which we're treating in issue #190.